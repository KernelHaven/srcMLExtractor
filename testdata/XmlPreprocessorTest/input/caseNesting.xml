<unit language="C">
	<switch>
		<block>
			<case>case 1:</case>
			<empty_stmt>S1</empty_stmt>
			<break>break;</break>
			<case>case 2:</case>
			<case>case 3:</case>
			<empty_stmt>S2</empty_stmt>
			<break>break;</break>
			<default>default:</default>
			<empty_stmt>S3</empty_stmt>
		</block>
	</switch>	
</unit>
