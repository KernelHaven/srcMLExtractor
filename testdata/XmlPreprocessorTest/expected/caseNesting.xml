<unit language="C">
	<switch>
		<block>
			<case>case 1:
				<empty_stmt>S1</empty_stmt>
				<break>break;</break></case>
			<case>case 2:</case>
			<case>case 3:
				<empty_stmt>S2</empty_stmt>
				<break>break;</break></case>
			<default>default:
				<empty_stmt>S3</empty_stmt></default>
		</block>
	</switch>	
</unit>
